_posts/2020-06-01-Map-Submissions.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Map submissions
+author: KingOfSquares
+---
+
+For a long time we have discussed opening up new map submissions, and now it is finally happening!
+
+The submission window is going to be open indefinitely, so no worries about submitting your map super fast. This post will go through the submission process, and will also include a template you will need to use at the bottom. Submissions that do not meet the requirements listed in this post and in the template will be closed, so be sure to read everything! 
+
+# The submission process
+
+Submissions will be done [here](https://github.com/OvercastCommunity/CommunityMaps/issues). To make a new submission click "New issue"
+
+#### The title of the post should be in the following form
+
+![Map title example](https://imgur.com/rTtjgK4.png)
+
+e.g
+
+![Map title example2](https://imgur.com/01Lfft7.png)
+
+#### For the post itself, please use the following template
+
+TEMPLATE
+
```

Comment:
No need for this since we have this https://github.com/OvercastCommunity/CommunityMaps/blob/master/.github/ISSUE_TEMPLATE/submission.md

Reply:
Hey, dont pretend like its always been there 😆 . Ill rewrite

Reply:
@KingOfSquares I just mean there's no need for a copy paste template when Issue Templates exist haha.

Reply:
No worries lol, just updated the post. Please read through!